Lib/keyword.py

```diff
@@ -0,0 +1,94 @@
+#! /usr/bin/python3.6
+
+"""Keywords (from "graminit.c")
```

Comment:
Note, this module we also have as a rust module. I prefer the rust module, since it picks the keywords directly from the lexer, so it's not a copy of the keywords.